### PR TITLE
docs: add FAQ entry about subpixel anti-aliased text

### PR DESCRIPTION
**Pull request by heronils**
#### docs: Add FAQ entry about subpixel anti-aliased text

Added an entry to docs/faq.md, which explains the reason for blurry looking fonts: A transparent font layer. It further explains how to fix it: By setting `backgroundColor` to a non-transparent color in the constructor of BrowserWindow. This will enable subpixel anti-aliased text and the font will stop looking blurry on LCD screens.

Resolves: #10955

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

**Comment by crypto480** [comment]
Quit doing dope

File: docs/tutorial/blurry-font.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Fixing Blurry Fonts
+
+If fonts look blurry for you, set the window background to a non transparent color.
+Do this in the constructor for [BrowserWindow][browser-window]:
+
+    new BrowserWindow({width: 1000, height: 750, backgroundColor: '#fff'});
```
Comment (malept) [inline docs/tutorial/blurry-font.md]: This should be in a fenced Markdown code block (for JavaScript).

File: docs/tutorial/blurry-font.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    //                                           ^ here                ^
+
+The reason for why the font may look blurry is that [sub pixel anti-aliasing](http://alienryderflex.com/sub_pixel/)
+will be deactivated when the background is transparent. See [this issue](https://github.com/electron/electron/issues/6344#issuecomment-420371918).
+
+Screenshot of the result:
+
+![subpixel rendering example](https://i.imgur.com/CjNQaeR.gif)
```
Comment (malept) [inline docs/tutorial/blurry-font.md]: We probably don't want to use linked images from imgur.

**Comment by heronils** [comment]
Hello @malept Sure, let this be a FAQ entry, with or without image. Please rewrite it / remove what you dont like.

For the records, this is how the official API docs look with and without the 'Hack':

electron api demos

**Comment by malept** [comment]
If you'd like to move the docs to a FAQ entry, the document is at `docs/faq.md`.

**Comment by heronils** [comment]
Ok, im just trying to do this.

**Comment by heronils** [comment]
OK, that should be good now.

**Comment by heronils** [comment]
Sorry, cant make release bot happy, it seems.

**Comment by jkleinsc** [comment]
@heronils The release-notes bot is unhappy because you deleted the issue template we use, specifically:
#### Description of Change


#### Checklist


- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).


#### Release Notes

Notes:

**Comment by heronils** [comment]
Well, i tried :-)

**Comment by malept** [comment]
> i inserted this text, still no success, what else am i missing?

Edit https://github.com/electron/electron/pull/16459#issue-246023306 with the [pull request template](https://github.com/electron/electron/blob/master/.github/PULL_REQUEST_TEMPLATE.md), filling it out as appropriate.

**Comment by heronils** [comment]
@malept thanks for helping me out again, did this. Now build-linux and build-mac-fork-prs still fails but this has nothing to do with my changes, as i did not change any code files. I just added an image to docs/images and i changed docs/faq.md.

**Comment by miniak** [comment]
@heronils release notes are only meant for user visible changes (APIs, bug fixes, etc.), updated docs should not be mentioned.

**Comment by jkleinsc** [comment]
@heronils I closed this PR in an attempt to get our lint process to verify the changes by closing and reopening the PR, but it appears that the repository is no longer available.  Feel free to reopen this PR if you still have the repository.

**Comment by jkleinsc** [comment]
Reopened this PR in #18101.

**Pull request closed without merging**